netbout-web/src/main/java/com/netbout/dynamo/DyAttachment.java

```diff
@@ -188,40 +187,7 @@ public InputStream read() throws IOException {
     @Override
     public void write(final InputStream stream, final String ctype,
         final String etag) throws IOException {
-        if (etag.isEmpty()) {
```

Comment:
@dmzaytsev TkAttach is the place where ctag and etag are created. It's strange to create them and check straight away. File content and name is something external - so it's the only thing we need to validate